test/integration/concierge_webhookauthenticator_status_test.go

```diff
@@ -0,0 +1,296 @@
+// Copyright 2024 the Pinniped contributors. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package integration
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"go.pinniped.dev/generated/latest/apis/concierge/authentication/v1alpha1"
+	"go.pinniped.dev/test/testlib"
+)
+
+func TestConciergeWebhookAuthenticatorStatus_Parallel(t *testing.T) {
+	testEnv := testlib.IntegrationEnv(t)
+	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Minute)
+	t.Cleanup(cancel)
+
+	tests := []struct {
+		name string
+		run  func(t *testing.T)
```

Comment:
Does this test table need to have a generic `run` function? Seems like they are all doing the same steps, so it could instead have fields like:
- The spec for a web hook authenticator
- The excepted phase for the authenticator
- The expected conditions for the authenticator

Reply:
It definitely could. I stuck to the same pattern as the other `_status_test.go` files in this directory (JWTAuthenticator, FederationDomain).  But I bet the JWTAuthenticator and WebhookAuthenticator both could just have a generic run instead.

Reply:
It's a weird style to have a test table that only has a run function. That kinda defeats the purpose of having a test table, imho. All these tests are almost running the same code, so wouldn't it be easy to convert it to a more standard looking test table, or am I missing something?

Reply:
Not a strong opinion here, I'll update!